Client/Frontend/Login Management/LoginListViewController.swift

```diff
@@ -330,8 +330,12 @@ extension LoginListViewController: SearchInputViewDelegate {
     }
 
     private func searchLoginsWithText(text: String) -> Success {
-        activeLoginQuery = profile.logins.searchLoginsWithQuery(text)
-            .bindQueue(dispatch_get_main_queue(), f: reloadTableWithResult)
+        activeLoginQuery =
+            (text.isEmpty ?
+                profile.logins.getAllLogins()
+                    .bindQueue(dispatch_get_main_queue(), f: reloadTableWithResult) :
```

Comment:
Gentle preference for phrasing this as:

```
let deferred = text.isEmpty ? profile.logins.getAllLogins() : profile.logins.searchLoginsWithQuery(text)
return deferred.bindQueue(dispatch_get_main_queue(), f: reloadTableWithResult)
```

but why not just make `SQLiteLogins.searchLoginsWithQuery` fall through to `getAllLogins` when passed an empty string?